api/src/schedules/tus.ts

```diff
@@ -0,0 +1,27 @@
+import { RESUMABLE_UPLOADS } from '../constants.js';
+import { getSchema } from '../utils/get-schema.js';
+import { createTusServer } from '../services/tus/index.js';
+import { scheduleSynchronizedJob, validateCron } from '../utils/schedule.js';
+
+/**
+ * Schedule the tus cleanup
+ *
+ * @returns Whether or not telemetry has been initialized
+ */
+export default function schedule() {
```

Comment:
I am not opposed to it, although where are you not seeing the inference? It infers it fine for me in all relevant locations.

Reply:
currently it's all good. we can also leave it as it. but having an explicit return time seems always more save to me in the long run.

currently one would be able to add `return;` in the middle of the function and TS wouldn't notice at "compile time" that consumers of the functions now cannot expect a boolean to be returned. this then could lead to a runtime error at some point. with an explicit return type, this then wouldn't happened. now this specific function is very small and _probably_ one would notice (although the name does not indicate that it will return a boolean). but still, I'd always prefer to have some sort of mechanism which prevents this right away. it's basically murphy's law, or like the first quote [here](https://www.howtocodeit.com/articles/ultimate-guide-rust-newtypes#newtype-essentials) 😄

Reply:
Fair point, added via 0d0009ec5083865698586370d2b5a5163ff6eaf6